packages/blue-sdk-ethers/src/fetch/Holding.ts

```diff
@@ -126,7 +122,7 @@ export async function fetchHolding(
         runner,
       )
         .isWhitelisted(user, overrides)
-        .catch(() => undefined);
+        .catch(() => false);
```

Comment:
We don't want to allow everyone by default in case of rpc call fail?

Reply:
Yeah that's what we wanted. And in this case we have no way of knowing the call failed... But at the same time it seems unsafe to me to behave differently than when the call returns `false` if the call fails because of the call fails offchain, it will fail onchain so it won't work. Except if the spec changes without upfront warning - which happened before and was the reason we introduced this change

So I'll introduce undefined back again